backend/data/src/test/kotlin/io/tolgee/util/WordCounterTest.kt

```diff
@@ -20,5 +20,6 @@ internal class WordCounterTest {
     assertThat(WordCounter.countWords("你好，这是一个简体中文文本。", "zh-Hans")).isEqualTo(7)
     assertThat(WordCounter.countWords("Hello, I am fred. (Super friend!)", "en-US")).isEqualTo(6)
     assertThat(WordCounter.countWords("Hello, I am fred. <html>yep!</html>", "en-US")).isEqualTo(7)
+    assertThat(WordCounter.countWords("What about {var_ids-var-ids}", "en-US")).isEqualTo(5) // 3 or 2?
```

Comment:
Well, I kept it simple, since the ICU messages can be complex, including escaping and stuff. Also, using variables doesn't happen too often so it doesn't affect the result so much.